Shared/Sources/AutoResizingMaskViewSource.swift

```diff
@@ -18,20 +19,37 @@ final class AutoResizingMaskViewSource {
 
   // MARK: Private
 
-  private static let padding: CGFloat = 20
+  private static let innerPadding: CGFloat = 20
   private static let cornerRadius: CGFloat = 2
   private static let borderWidth: CGFloat = 1
   private static let borderColor = UIColor.borderColor().colorWithAlphaComponent(0.5).CGColor
   private static let defaultViewSideLength: CGFloat = 50
   private static let leftRightWidth: CGFloat = 80
 
-  private let origin: CGPoint
+  private let outerOrigin: CGPoint
+  private let innerOrigin: CGPoint
   private let flexibleWidthHeightSize: CGSize
   private let flexibleTopBottomSize: CGSize
 
+  private static var horizontalPadding: CGFloat = {
```

Comment:
One technique to avoid having to use `self.dynamicType` to make reference to "class constants" like this is to create an inner struct type.

Eg:
```
struct Constants {
   static let horizontalSpacing = 10
   static let verticalSpacing = 20
}

```

Then using it later by referencing the type `Constants.horizontalSpacing`.  Obviously in this case there would be the added complexity of `#if os(tvOS)` etc.